src/main/java/com/battleship/backend/controllers/GameController.java

```diff
@@ -0,0 +1,87 @@
+package com.battleship.backend.controllers;
+import com.battleship.backend.BoardRepository;
+import com.battleship.backend.exceptions.GameHasAlreadyStartedException;
+import com.battleship.backend.exceptions.GameNotReadyToStartException;
+import com.battleship.backend.exceptions.InvalidHitException;
+import com.battleship.backend.exceptions.InvalidShipPlacementException;
+import com.battleship.backend.models.*;
+import com.battleship.backend.validators.HitRequestValidator;
+import com.battleship.backend.validators.PlaceShipsValidator;
+import com.battleship.backend.validators.Validator;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+@Controller
+public class GameController {
+    Validator placeShipsValidator;
+    Validator hitRequestValidator;
+    Game game;
+
+
+    public GameController(Game game){
+        this.game = game;
+        this.placeShipsValidator = game.getPlaceShipsValidator();
+        this.hitRequestValidator = game.getHitRequestValidator();
+
+    }
+
+    @GetMapping("/boards")
+    public @ResponseBody
+    Boardable[] getBoards() {
+        return game.getBoards();
+    }
+
+    @PatchMapping("/board/place")
+    public @ResponseBody
+    Boardable placeShips(@RequestBody Request placeRequest) throws Exception{
+        Boardable playerBoard = game.getPlayerBoard();
```

Comment:
Is the player placing one ship or a list of ships?

Reply:
It's supposed to be placeShip, it's a typo. Will fix.